src/libraries/System.Memory.Data/src/System/BinaryData.cs

```diff
@@ -0,0 +1,262 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System
+{
+    /// <summary>
+    /// A lightweight abstraction for a payload of bytes that supports converting between string, stream, JSON, and bytes.
+    /// </summary>
+    public class BinaryData
+    {
+        private const int CopyToBufferSize = 81920;
```

Comment:
Nit: I think this would be better as a const inside of the method where it's used, and with a comment that it matches the value used implicitly by CopyTo{Async}